src/node_link.cc

```diff
@@ -4,15 +4,20 @@
 
 #include "node_link.h"
 
+namespace {
+
+double DEFAULT_QUANTUM = 4.0;
```

Comment:
Note: I've moved this into a constant and reused the same quantum for requestBeat/forceBeat that we already use elsewhere